src/layer/vector/Polyline.js

```diff
@@ -172,6 +172,21 @@ export var Polyline = Path.extend({
 		return this.redraw();
 	},
 
+	// @method setLastLatLng(latlng: LatLng, latlngs? LatLng[]): this
+	// Modifies the last point of the polyline to be the specified point.
+	// By default, adds to the first ring of the polyline in case of a
+	// multi-polyline, but can be overridden by passing a specific ring as a
+	// LatLng array (that you can earlier access with
+	// [`getLatLngs`](#polyline-getlatlngs)).
+	setLastLatLng: function (latlng, latlngs) {
+		latlngs = latlngs || this._defaultShape();
+		latlng = toLatLng(latlng);
+		latlngs.pop();
```

Comment:
This doesn't cover the edge case of a polyline without any `latlng`s. `var line = L.polyline([]);` is a valid thing to do.

Reply:
It does :) it is precisely my use-case (I start with an empty polyline and add points along the way). If you test the following in your browser:
```>> a=[]
Array []
>> a.pop()
undefined```
You can see that popping an empty list is no problem.